Query.php

```diff
@@ -607,4 +611,20 @@ public function minScore($minScore)
         $this->minScore = $minScore;
         return $this;
     }
+
+    /**
+     * Sets the options to be passed to the command created by this query.
```

Comment:
should be more clear about what kind of options these are, e.g. `options that are appended to the query URL such as preference, ... etc...`